tardis/io/decay.py

```diff
@@ -85,7 +84,7 @@ def decay(self, t):
 
         Returns
         -------
-            : decayed abundances
+        pandas.DataFrame
```

Comment:
```suggestion
        pandas.DataFrame
            Decayed abundances
```
That was description which was wrongly written as type, let's move it down.